src/components/Set/Summary/index.tsx

```diff
@@ -0,0 +1,205 @@
+  if (!literature) return null;
+  const literatureEntries: [string, Reference][] = literature.map((ref) => {
+    const journalRegExp = /(.+) (\d{4});(\d+):(\d+-\d+)./;
+    const matches = journalRegExp.exec(ref.journal || '');
+    if (matches) {
+      return [
+        String(ref.PMID),
+        {
+          ...ref,
+          ISO_journal: matches[1],
+          year: Number(matches[2]),
+          volume: matches[3],
+          raw_pages: matches[4],
+        } as Reference,
+      ];
+    }
+    return [String(ref.PMID), ref];
+  });
+  return (
+    <>
+      <h4>Literature</h4>
+      <Literature extra={literatureEntries} />
+    </>
+  );
+};
+
+const SetDescription = ({
+  accession,
+  description,
+}: {
+  accession: string;
+  description: string;
+}) => {
+  if (!accession || !description) return null;
+  return (
+    <>
+      <h4>Description</h4>
+      <Description textBlocks={[description]} accession={accession} />
+    </>
+  );
+};
+
+const SetAuthors = ({ authors }: { authors: Array<string> }) => {
+  if (!authors) return null;
+  return (
+    <tr>
+      <td>Authors</td>
+      <td data-testid="set-type">{authors.join(', ')}</td>
+    </tr>
+  );
+};
+
+type Props = {
+  data: {
+    metadata: SetMetadata;
+  };
+  loading: boolean;
+};
+
+const SummarySet = ({ data, loading }: Props) => {
+  const metadata: SetMetadata =
+    loading || !data.metadata
+      ? {
+          accession: '',
+          name: { name: '' },
+          description: '',
+          id: '',
+          source_database: '',
+          authors: null,
+          literature: [],
+          counters: {},
+          relationships: {
+            nodes: [],
+            links: [],
+          },
+        }
+      : data.metadata;
+  let currentSet = null;
+  if (metadata.source_database) {
+    for (const db of setDBs) {
+      if (db.name === metadata.source_database) currentSet = db;
+    }
+    if (metadata.source_database === 'panther')
+      metadata.description = metadata.name.name;
+  }
+
+  return (
+    <div className={css('vf-stack', 'vf-stack--400')}>
+      {metadata.relationships &&
+        metadata.relationships.nodes &&
+        metadata.relationships.nodes.map((m) => (
+          <SchemaOrgData
+            key={m.accession}
+            data={{ data: m, db: metadata.source_database }}
+            processData={schemaProcessData}
+          />
+        ))}
+      <section className={css('vf-grid', 'summary-grid')}>
+        <div className={css('vf-stack')}>
+          <table className={css('vf-table', 'left-headers')}>
+            <tbody>
+              <tr>
+                <td style={{ maxWidth: '50%' }}>Accession</td>
+                <td data-testid="set-accession">
+                  <Accession accession={metadata.accession} />
+                </td>
+              </tr>
+              <tr>
+                <td>Data type</td>
+                <td data-testid="set-type">Set</td>
+              </tr>
```

Comment:
```suggestion
```

That might be an artifact from when _sets_ were going to be clans, pathways, and other data types. I don't think we need this row.